app/src/main/java/com/example/kotlincalculator/CalculatorActivity.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.kotlincalculator
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import com.example.kotlincalculator.mvp.model.CalculatorModel
+import com.example.kotlincalculator.mvp.presenter.CalculatorPresenter
+import com.example.kotlincalculator.mvp.view.CalculatorView
+import kotlinx.android.synthetic.main.activity_main.button_zero
+import kotlinx.android.synthetic.main.activity_main.button_one
+import kotlinx.android.synthetic.main.activity_main.button_two
+import kotlinx.android.synthetic.main.activity_main.button_three
+import kotlinx.android.synthetic.main.activity_main.button_four
+import kotlinx.android.synthetic.main.activity_main.button_five
+import kotlinx.android.synthetic.main.activity_main.button_six
+import kotlinx.android.synthetic.main.activity_main.button_seven
+import kotlinx.android.synthetic.main.activity_main.button_eight
+import kotlinx.android.synthetic.main.activity_main.button_nine
+import kotlinx.android.synthetic.main.activity_main.button_sum
+import kotlinx.android.synthetic.main.activity_main.button_multiply
+import kotlinx.android.synthetic.main.activity_main.button_share
+import kotlinx.android.synthetic.main.activity_main.button_subtract
+
+class CalculatorActivity : AppCompatActivity(), View.OnClickListener {
```

Comment:
do not use the activity as a listener please

Reply:
done